QUEST_LV_0200.tsv

```diff
@@ -7674,18 +7674,18 @@ QUEST_LV_0200_20150714_007677	$I paid a lot for this at the black market in Fedi
 QUEST_LV_0200_20150714_007678	This is sturdy. You've been suffered by the trap of Evoniphon.{nl}But, what have brought you here?{nl}
 QUEST_LV_0200_20150714_007679	You are the Revelator? It's so nice to meet you.{nl}Thanks for saving me... I have another request for you.
 QUEST_LV_0200_20150714_007680	We are doing an important mission at the moment.
-QUEST_LV_0200_20150714_007681	This is a little complicated.{nl}We are chasing after Evoniphon who is the suspect of the murder case at the Star Tower.{nl}But, he is being protected by these religious rules at the monastery nearby.{nl}
+QUEST_LV_0200_20150714_007681	This is a little complicated.{nl}We are chasing after Evoniphon who is the prime suspect of the murder case at the Star Tower.{nl}But, he is being protected by these religious rules at the monastery nearby.{nl}
 QUEST_LV_0200_20150714_007682	So, please help us to get Evoniphon.{nl}We should somehow first persuade the monks.. Though, that's going to be tricky.
 QUEST_LV_0200_20150714_007683	I am so sorry. {nl}Please first persuade Potos at Diliti Court.
 QUEST_LV_0200_20150714_007684	The monks shake their heads in denial when we mention Evoniphon..
 QUEST_LV_0200_20150714_007685	Do you know of the Star Tower?{nl}Starting with Lydia Schaffen, the greatest archers are all originated from that place.{nl}
-QUEST_LV_0200_20150714_007686	The head of the Star Tower was killed by Evoniphon.{nl}I've chased him up till here, but I'm having trouble of doing anything further,{nl}because of the religious rules of the sanctum.{nl}
+QUEST_LV_0200_20150714_007686	The head of the Star Tower was killed by Evoniphon.{nl}I've chased him up till here, but I'm having trouble of doing anything further,{nl}because of the religious rules of the monastery.{nl}
```

Comment:
Korean says sanctum/sanctuary. Probably the latter since I see 'sanctuary' being used more.